fragdenstaat_de/theme/templatetags/fds_tags.py

```diff
@@ -23,4 +23,8 @@ def needs_glyphosat_attachment(message):
 def needs_frontex_import(message, user):
     if not user.is_staff and not user.is_superuser:
         return False
+
+    tags = set(message.tags.all().values_list("name", flat=True))
```

Comment:
There is a `tag_set` cached property on messages that you can use. This avoids getting tags for a message multiple times. (That property could actually be improved by using `values_list` like here.)